tests/test_fields.py

```diff
@@ -522,3 +523,110 @@ def b(self):
             return None
 
     assert set(get_fields_dict(B)) == {"a", "b", "mixin"}
+
+
+@pytest.mark.asyncio
+async def test_field_disabled() -> None:
+    @attrs.define
+    class Item:
+        x: int
+        y: Optional[int] = None
+        z: Optional[int] = None
+
+    class Page(ItemPage[Item]):
+        @field
+        def x(self) -> int:
+            return 1
+
+        @field(disabled=False)
+        def y(self) -> int:
+            return 2
+
+        @field(disabled=True)
```

Comment:
Maybe `enabled` would be a better name.

I would prefer something that suggests how the field can still be used as a property but is not included in the `to_item` output, but I cannot think of a good name (`export`?  `itemize`? `itemizable`? `to_item`? `add_to_item`? `include_in_item`?).

Reply:
My initial implementation actually used `enabled`, but it seems `get_fields_dict(include_disabled=True)` sounds much better.

Hmmm, `to_item` and `include_in_item` does sound better in terms of what it does. Currently, either `enabled` or `disabled` isn't that clear about what it does.

+1 on something like `@field(to_item=False)`.

Reply:
I was initially thinking about something like "exclude_by_default=True" or "excluded_by_default=True", or "extract_by_default=False"; alternatives like "to_item=False" also look fine. 

A possible issue with to_item=False is that you may think such field doesn't have to be defined in the item, but it does; I think it's still an error if item doesn't have it.

Reply:
> I was initially thinking about something like "exclude_by_default=True" or "excluded_by_default=True", or "extract_by_default=False"

I think these have the same issue with `include` and `exclude` where it's not clear what's happening, i.e., where it's being excluded from.

> A possible issue with to_item=False is that you may think such field doesn't have to be defined in the item, but it does; I think it's still an error if item doesn't have it.

You're right. What do you think about having something like `omit_in_to_item=True` or `to_item_omit=True`?